pkg/apic/subscriptionmanager.go

```diff
@@ -223,17 +283,22 @@ func (sm *subscriptionManager) Start() {
 	if !sm.isRunning {
```

Comment:
just in general, for the naming below, what does "uc", "acc", and "ar" stand for?

Reply:
added comments where these are declared